src/infos.rs

```diff
@@ -10,3 +30,73 @@ static API_LEVEL: u8 = if cfg!(target_os = "nanos") {
 } else {
     0xff
 };
+
+#[used]
+#[link_section = "ledger.api_level"]
+static ELF_API_LEVEL: [u8; 4] = if cfg!(target_os = "nanos") {
+    *b"0\n\0\0"
+} else if cfg!(target_os = "nanox") {
+    *b"5\n\0\0"
+} else if cfg!(target_os = "nanosplus") {
+    *b"1\n\0\0"
+} else {
+    *b"255\n"
+};
```

Comment:
Should be update now that API_LEVEL is now at build time from SDK?

Reply:
Done in https://github.com/LedgerHQ/ledger-device-rust-sdk/pull/96